newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -149,13 +150,24 @@ export default class LocalPreviewLauncher extends React.Component<
         new gd.AbstractFileSystemJS(),
         localFileSystem
       );
-      const outputDir = path.join(fileSystem.getTempDir(), 'preview');
-      const exporter = new gd.Exporter(fileSystem, gdjsRoot);
-
-      return {
-        outputDir,
-        exporter,
-      };
+      try {
+        const outputDir = path.join(fileSystem.getTempDir(), `preview`);
+        const exporter = new gd.Exporter(fileSystem, gdjsRoot);
+        return {
+          outputDir,
+          exporter,
+        };
+      } catch (error) {
+        const outputDir = path.join(
+          fileSystem.getTempDir(),
+          `preview-${os.userInfo().uid}`
```

Comment:
You also inverted the catch and the try. But again, there is no need for uid here and no catch. Revert the changes back the how it was before the PR :)

Reply:
OK

Reply:
Done